Dockerfile

```
@@ -7,6 +7,21 @@ RUN chmod +x ./gradlew && ./gradlew clean bootJar -x test
+# 앱 JAR 복사
 COPY --from=build /app/build/libs/app.jar app.jar
+
+# OpenTelemetry Java Agent 추가
+ADD https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar /otel/opentelemetry-javaagent.jar
+
 EXPOSE 8080
-ENTRYPOINT ["java","-jar","app.jar"]
+
+# OpenTelemetry Agent 적용된 실행
+ENTRYPOINT ["java",
+            "-javaagent:/otel/opentelemetry-javaagent.jar",
```

Hadolint (2.12.0)

[error] unexpected '"'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)